documentation/README.md

```diff
@@ -0,0 +1,45 @@
+# To-do app
+## About the application
+
+This application is a web-browser-based to-do app, written in Python utilising the Flask web development framework. Data is stored in a Trello board.
+<br><br>
+
+## Config
+
+Environment variables are read in from the `.env` file by Docker at runtime. This file is not copied to the container, nor is it committed to the git repo, so you will need to create it if it doesn't exist. You can use `.env.template` as a template.
+
+Environment variables include:
+- Flask server configuration variables.
+- Trello authorisation parameters
+<br><br>
+
+## Running the application
```

Comment:
Wait, can I run the application normally as well (e.g. if I wanted to debug)? What would I have to do in that case?